driver.c

```diff
@@ -142,8 +133,7 @@ int main(int argc, char **argv)
         break;
     }
     default:
-        printf("Abnormal: unknown request: %d\n", req);
-        return -1;
+        FATAL(-1, "Abnormal: unknown request: %d\n", req);
         break;
```

Comment:
I agree the 'break' is not a must and logically useless.

But I'm not very confident in this change since I met not so smart compiler/static analyzer before, which warns the break is missing.

Also, it is a generally a good practice to break to match case keep it there in case someone changes the line to, for example, force it to do a print instead of letting the macro do it. In this case, an accidental fall-through happens.